docs/quickstart.rst

```diff
@@ -105,22 +111,35 @@ These quick functions give you slightly less control over the machine learning w
 Walkthrough
 -----------
```

Comment:
Created examples/walkthrough.ipynb to ensure that this code works correctly. It might be nice to put a binder link here in the future.